urns/urns.go

```diff
@@ -86,6 +90,16 @@ func NewTelegramURN(identifier int64, display string) URN {
 	return NewURNFromParts(TelegramScheme, strconv.FormatInt(identifier, 10), display)
 }
 
+// NewWhatsAppURN returns a URN for the passed in whatsapp identifier
+func NewWhatsAppURN(identifier string) (URN, error) {
+	// validate identifier
+	urn := NewURNFromParts(WhatsAppScheme, identifier, "")
+	if !urn.Validate() {
```

Comment:
occurs to me that `Validate` really should return an error.. but I'll do that in another PR